known-issues.yml

```diff
@@ -144,6 +144,15 @@ zib-DrugUse:
    Observation.component:routeOfAdministration.value[x]:
       - cardinality: 0..1
         reason: The cardinality mismatch between the zib (0..*) and FHIR (0..1) is explained by the parent .component element that may exists 0..* times, thereby this mapping is still honoring the cardinality requirements of the zib.
+        
+zib-FLACCpainScale:
+  zib deviations:
+   Observation:
+      - short: FLACCpainScale
+        reason: The root concept name 'FLACCpainScore' is an mistake in the zib and will be fixed at the next pre-publication.
+   Observation.note.text:
```

Comment:
Can you merge main into this branch and update the known-issues.yml accordingly?

Reply:
Fix!